virus/gladius_clandestinus.md

```diff
@@ -0,0 +1,177 @@
+# Hijacking our attention: characterization of a viroid-like agent that may act as a satellite of an E.coli virus
+written by: Clare Gillis (https://github.com/claregillis)
+
+### *Gladius clandestinus*
+
+[Optional: Tutorial 2 Generative AI Cover-art](img/<virusname>/cover.png)
```

Comment:
Please amend the Image path to "point" to your images, you can refer to the markdown code in the other students examples.